src/Domain/Tags/AreaElement.hpp

```diff
@@ -29,7 +30,7 @@ namespace domain::Tags {
  * metric.
  */
 template <typename SourceFrame, typename TargetFrame>
-struct DetSurfaceJacobian : db::SimpleTag {
+struct AreaElement : db::SimpleTag {
```

Comment:
tl;dr I suggest this:

- Keep this tag unchanged (no rename, no change in what it stores).
- Include the metric determinant in your `area_element` functions so they actually integrate to the area in curved space.

---

Details: I think in curved space we should make the distinction between the surface Jacobian determinant `J` and the area element `\sqrt{gamma} J`. The area element should actually integrate to the area, hence it should include the metric determinant. Arguably the surface Jacobian alone (without the metric determinant) isn't much use, but you probably don't want to go through the code that uses the `Tags::DetSurfaceJacobian` and make sure it uses the metric determinant properly. Therefore, just keep this tag unchanged. But in your `area_element` functions include the metric determinant, and check in the test that you get the horizon area of Kerr by just integrating over the area element.

Reply:
I don't have a strong opinion but this is then inconsistent with `area_element` in `StrahlKorperGr.cpp` from what I can tell.

Reply:
In the StrahlkorperGr test here https://github.com/sxs-collaboration/spectre/blob/develop/tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp#L417 the integral over the area element is the area, without an additional metric determinant. It follows Appendix D in https://arxiv.org/abs/gr-qc/9606010, which is explained in a bit more detail in Appendix C in the Baumgarte/Shapiro book. That's what we should do for this `area_element` function as well. I think multiplying by the spatial metric determinant is enough.

Reply:
ah right, that makes sense. I changed it so